site/content/ecosystem/arangodb-mcp-server.md

```diff
@@ -0,0 +1,322 @@
+---
+title: ArangoDB Model Context Protocol (MCP) Server
+menuTitle: MCP Server
+weight: 10
+description: >-
+  A Model Context Protocol server for generating and executing AQL queries using AI assistants like Claude and Cursor IDE
+---
+The ArangoDB MCP Server is a focused [Model Context Protocol](https://modelcontextprotocol.io/) (MCP) implementation that enables AI assistants to generate and execute AQL queries based on natural language questions. It includes lightweight schema discovery and manuals to ground queries in actual database structure.
+
+## Features
+
+**AQL Generation & Execution:**
+- Generate AQL grounded in actual database structure
+- Execute AQL with optional bind variables and target database
+
+**Manuals for Guidance:**
+- AQL reference and optimization guides built-in
+- Context-aware query generation
+
+**Lightweight Schema Discovery:**
+- List collections within accessible databases
+- Sample documents via simple filters to learn fields
+
+## What You Can Do
+
+The server is purpose-built for safe, read-focused AQL operations:
+- Execute AQL queries with optional bind variables and target database
+- Access built-in manuals for syntax and optimization guidance
+- Discover database schemas and collection structures
+- Sample documents to understand field structures
+
+The following are not included:
+- Graph/view/index/analyzer management tools
+- Destructive admin operations (create/delete databases or collections)
+
+## Getting Started
+
+Choose the setup that works best for you. Docker is recommended for quick start with everything bundled together.
+
+### Option 1: Docker Setup (Recommended)
+
+This approach bundles the MCP server and ArangoDB together, perfect for testing and development.
+
+**Prerequisites:**
+- Docker installed
+- Cursor IDE or Claude Desktop
+
+1. Build the MCP server image:
+
+```bash
+cd mcp-arango-aql
+docker build -t arangodb-mcp-server:dev -f Dockerfile.dev .
+```
+
+2. Start ArangoDB (if you don't have an instance):
+
+```bash
+docker run -d --name arangodb -p 8529:8529 -e ARANGO_ROOT_PASSWORD=test arangodb/arangodb:latest
```

Comment:
Should use image `arangodb` or `arangodb/enterprise`. We no longer publish `arangodb/arangodb` (CE) since 3.12.5